scheduler/src/cook/mesos/api.clj

```diff
@@ -1369,27 +1370,51 @@
       (when-let [uncoerced-query-params (get-in ctx [:request :query-params "job"])]
         (mapv #(UUID/fromString %) (vectorize uncoerced-query-params)))
       (get-in ctx [:request :body-params :jobs])
-      [(get-in ctx [:request :body-params :job])]))
+      (vectorize (get-in ctx [:request :body-params :job]))))
 
 (defn check-jobs-exist
   [conn ctx]
-  (let [jobs (jobs-from-request ctx)
+  (let [jobs (or (::jobs ctx) (jobs-from-request ctx))
         jobs-not-existing (remove (partial job-exists? (d/db conn)) jobs)]
-    (if (seq jobs-not-existing)
+    (cond
+      ; error case
+      (seq jobs-not-existing)
       [false {::error (->> jobs-not-existing
                            (map #(str "UUID " % " does not correspond to a job."))
                            (str/join \space))}]
-      [true {::jobs jobs}])))
+      ; if the context doesn't already have ::jobs, add it
```

Comment:
Nitpick: I would favor a newline separating the condition/action pairs

Reply:
Done.